geode-core/src/main/java/org/apache/geode/management/internal/ManagementAgent.java

```diff
@@ -196,7 +196,8 @@ private void loadWebApplications() {
     }
 
     try {
-      if (agentUtil.isAnyWarFileAvailable(adminRestWar, pulseWar)) {
+      HttpService httpService = cache.getHttpService().get();
+      if (httpService != null && agentUtil.isAnyWarFileAvailable(adminRestWar, pulseWar)) {
```

Comment:
If `cache.getHttpService().get()` doesn't throw `NoSuchElementException` then, by definition, `httpService` is `not null`, so the condition `httpService != null` is redundant.

Reply:
Ah - thanks! Fixed that and the prior comment too.